### ISSUE TITLE
Feature suggestion - offline mode

### ISSUE THREAD

**Issue by oyvindbusk** [brentp/seqcover#25]
It would be nice to be able to run this on servers with restricted internet access. Would it be possible to have a solution where you could generate a file with transcript info that can be reused in these cases?

```
# Something like this to generate the file:
seqcover generate-db --genes PIGA,KCNQ2,ARX,DNM1,SLC25A22,CDKL5,GABRA1 \
		--hg19 \
		-o offline_db.json

# Then something like this to use it
seqcover report --genes PIGA,KCNQ2,ARX,DNM1,SLC25A22,CDKL5,GABRA1,CAD,MDH2,SCN1B,CNPY3,CPLX1,NEB,HNRNPA1,CCDC39,AIFM1,CHCHD10 \
		--background seqcover/seqcover_p5.d4 \
		--fasta $fasta samples/*.bed.gz \
		-r my_genes_report.html \
		--offline-db offline_db.json
```

**Comment by brentp** [comment]
but why not just generate the report? and then move it wherever you need?

you mean the machine where you have *.bed.gz does not allow accessing mygene.info?

**Comment by oyvindbusk** [comment]
I was considering adding seqCover to auto-generate a report batchwise as part of a variant calling pipeline for a small gene panel that we run routinely. The server that runs that pipeline is offline and cannot access mygene.info.

This might not be relevant for many people, so I will try to find a workaround

**Comment by brentp** [comment]
In your first seqcover call in the example, the offline_db would have to be a superset of genes from the report (2nd) step, right?

I think this should be do-able. happy to review a PR or I can try to implement after next week.

**Comment by oyvindbusk** [comment]
Whoops, I chose a bad example, that wouldn't work. Yes, the offline_db would have to contain all the the genes that are requested in the second step. 

I will try to make it work and send you a PR within a couple of days.

**Comment by brwnj** [comment]
resolved in 805cae7331fef0007814882431afb938df889e24.
